gravitee-apim-console-webui/src/management/api/proxy/endpoints/list/api-proxy-endpoint-list.component.ts

```diff
@@ -105,6 +105,40 @@ export class ApiProxyEndpointListComponent implements OnInit {
       .subscribe();
   }
 
+  deleteEndpoint(groupName: string, endpointName: string): void {
+    this.matDialog
+      .open<GioConfirmDialogComponent, GioConfirmDialogData>(GioConfirmDialogComponent, {
+        width: '500px',
+        data: {
+          title: 'Delete Endpoint',
+          content: `Are you sure you want to delete the Endpoint <strong>${endpointName}</strong>?`,
+          confirmButton: 'Delete',
+        },
+        role: 'alertdialog',
+        id: 'deleteEndpointConfirmDialog',
+      })
+      .afterClosed()
+      .pipe(
+        takeUntil(this.unsubscribe$),
+        filter((confirm) => confirm === true),
+        switchMap(() => this.apiService.get(this.apiId)),
+        switchMap((api) => {
+          _.remove(_.find(api.proxy.groups, (g) => g.name === groupName).endpoints, (e) => e.name === endpointName);
+          return this.apiService.update(api);
+        }),
+        catchError(({ error }) => {
+          this.snackBarService.error(error.message);
+          return EMPTY;
+        }),
+        tap((api) => {
+          this.initData(api);
+          this.ajsRootScope.$broadcast('apiChangeSuccess', { api: _.cloneDeep(api) });
```

Comment:
not necessary if is all-ready in the service ? 🤔 or not ?

Reply:
I think it's outdated. There is a dedicated commit where I remove all calls to the root scope normally.